po/Makefile.am

```diff
@@ -112,10 +112,8 @@ update-po: po/cockpit.pot
 $(WEBLATE_REPO):
 	git clone --depth=1 -b $(WEBLATE_REPO_BRANCH) $(WEBLATE_REPO_URL) $(WEBLATE_REPO)
 
-upload-pot: po/cockpit.pot $(WEBLATE_REPO)
-	cp $(builddir)/po/cockpit.pot $(WEBLATE_REPO)
-	git -C $(WEBLATE_REPO) commit -m "Update source file" -- cockpit.pot
-	git -C $(WEBLATE_REPO) push
+upload-pot:
```

Comment:
This is a hack until we adjust bots/po-refresh to not expect the target any more, right? And consequently fix podman/machines to separate POT uploads as well. I don't think we should keep a broken rule permanently.

Reply:
This is a compatibility shim for the other projects, yes.  I'm not ready to rip out the invocation of that rule from `bots` until the other projects also don't need it.

Reply:
Yes, of course. I do agree we need to keep it for a bit, I just wanted to ensure it's not going to stay like this forever.